### ISSUE TITLE
[Story] Join a room after request has been accepted

### ISSUE THREAD

**Issue by mxandreas** [vector-im/element-meta#2583]
### Description

* As a user whose request to join room X has been accepted
* I want to see an invitation to the room
* So that I can use it to join the room or decline the invite because I have changed my mind or this was a mistake

Both, from user and technical perspective, the invite that has been created as a result of accepting the request, works in the same was as an invite when the person is (proactively) invited.

### Acceptance criteria

- Accepting or declining the invite works in the same way as currently when the person is proactively invited

### Size estimate

None

### Dependencies

- None

### Out of scope

- Nothing

### Open questions

```[tasklist]
### Questions
```

### Subtasks

```[tasklist]
### Android
```

```[tasklist]
### iOS
```

```[tasklist]
### Rust
```

```[tasklist]
### Other
```

**Comment by ganfra** [comment]
@mxandreas I don't think there is anything to do here?

**Comment by mxandreas** [comment]
Yes, there is probably no code to be written here as we can "reuse" what is there already. However, I'd still have a story to check the box that this works, once the other pieces are there.

**Comment by ganfra** [comment]
@mxandreas should we close the issue? or we wait for final testing session?
